app/models/simulation.rb

```diff
@@ -7,4 +7,7 @@ class Simulation < ActiveRecord::Base
   has_many :content, class_name: 'Content'
 
   accepts_nested_attributes_for :users
+
+  has_one :name
+  has_one :requester
```

Comment:
I don't think will need the `attr_accessible` but the columns are correct, you can add them to the current migration reset the db.